apps/client/src/app/shop/[shopid]/_components/Icon/Icon.tsx

```diff
@@ -0,0 +1,11 @@
+import { LABEL_TO_ICON } from "./Icon.constants";
+
+import type { IconProps } from "./Icon.types";
+
+export const Icon = ({ label, fontSize, color }: IconProps) => {
+  return (
+    <span className="material-icons-outlined" style={{ fontSize, color }}>
+      {LABEL_TO_ICON[label]}
+    </span>
+  );
+};
```

Comment:
물론 아래 span에 넘겨주는 프로퍼티명은 fontSize지만 사용할 때는 size로 받아도 되지 않을까요?

```suggestion
export const Icon = ({ named, size, color }: IconProps) => {
  return (
    <span className="material-icons-outlined" style={{ fontSize: size, color }}>
      {named}
    </span>
  );
};
```